plugins/woocommerce-blocks/assets/js/blocks/product-filters/edit.tsx

```diff
@@ -128,8 +169,147 @@ export const Edit = ( {}: BlockEditProps< ProductFiltersBlockAttributes > ) => {
+							label={ __( 'Always', 'woocommerce' ) }
+						/>
+					</ToggleGroupControl>
+					{ attributes.overlay === 'mobile' && (
+						<>
+							<RadioControl
+								className="wc-block-editor-product-filters__overlay-button-style-toggle"
+								label={ __( 'Button', 'woocommerce' ) }
+								selected={ attributes.overlayButtonStyle }
+								onChange={ (
+									value: BlockAttributes[ 'overlayButtonStyle' ]
+								) => {
+									setAttributes( {
+										overlayButtonStyle: value,
+									} );
+								} }
+								options={ [
+									{
+										value: 'label-icon',
+										label: __(
+											'Label and icon',
+											'woocommerce'
+										),
+									},
+									{
+										value: 'label',
+										label: __(
+											'Label only',
+											'woocommerce'
+										),
+									},
+									{
+										value: 'icon',
+										label: __( 'Icon only', 'woocommerce' ),
+									},
+								] }
+							/>
+							{ attributes.overlayButtonStyle !== 'label' && (
+								<>
+									<ToggleGroupControl
+										className="wc-block-editor-product-filters__overlay-button-toggle"
+										isBlock={ true }
+										value={ attributes.overlayIcon }
+										onChange={ (
+											value: BlockAttributes[ 'overlayIcon' ]
+										) => {
+											setAttributes( {
+												overlayIcon: value,
+											} );
+										} }
+									>
+										<ToggleGroupControlOption
+											value={ 'filter-icon-1' }
+											label={
+												<Icon
+													size={ 32 }
+													icon={ filter }
+												/>
+											}
+										/>
+										<ToggleGroupControlOption
+											value={ 'filter-icon-2' }
+											label={
+												<Icon
+													size={ 32 }
+													icon={ filterThreeLines }
+												/>
+											}
+										/>
+										<ToggleGroupControlOption
+											value={ 'filter-icon-3' }
+											label={
+												<Icon
+													size={ 32 }
+													icon={ menu }
+												/>
+											}
+										/>
+										<ToggleGroupControlOption
+											value={ 'filter-icon-4' }
+											label={
+												<Icon
+													size={ 32 }
+													icon={ settings }
+												/>
+											}
+										/>
+									</ToggleGroupControl>
+									<RangeControl
+										label={ __(
+											'Icon size',
+											'woocommerce'
+										) }
+										className="wc-block-editor-product-filters__overlay-button-size"
+										value={ attributes.overlayIconSize }
+										onChange={ ( value: number ) =>
+											setAttributes( {
+												overlayIconSize: value,
+											} )
+										}
+										min={ 20 }
+										max={ 80 }
+									/>
+								</>
+							) }
+						</>
+					) }
+					<p className="wc-block-editor-product-filters__overlay-link">
+						<ExternalLink href={ templatePartEditUri }>
+							{ __( 'Edit overlay', 'woocommerce' ) }
+						</ExternalLink>
+					</p>
```

Comment:
This paragraph is styled as a button. Should we use another tag for it. Also, only the link is clickable, not the "button", so it may be confusing to users.

Reply:
![image](https://github.com/user-attachments/assets/2a59e491-d3f7-422f-9a95-6d91f14dcf90)